subsys/spm/spm.c

```diff
@@ -8,6 +8,7 @@
 #include <misc/printk.h>
 #include <device.h>
 #include <gpio.h>
+#include "spm_spu.h"
```

Comment:
Can we add an "internal" suffix/prefix to this one, to stress that this is just an internal inclusion?
(see how we do, for example, in upstream project: `arch/arm/core/cortex-m/mpu`)

Reply:
Renamed to spm_internal.h.

Added defines for magic numbers at the top of that file.